wolfSSL/src/chapter11.md

```diff
@@ -342,22 +343,6 @@ if ( (ssl = wolfSSL_new(ctx)) == NULL) {
 wolfSSL_set_fd(ssl, connfd);
 ```
 
-A WOLFSSL object needs to be created after each TCP Connect and the socket file descriptor needs to be associated with the session.
```

Comment:
Let's keep this sentence here. To be clear that a new object is needed per file descriptor

Reply:
The rest though can be deleted.